android/app/build.gradle

```diff
@@ -113,8 +113,8 @@ android {
         applicationId 'io.mosip.residentapp'
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionName androidGitVersion.name()
-        versionCode 1
+        versionName "0.15.0"
+        versionCode 15
```

Comment:
version code can't be hardcoded. It's get updated during publishing

Reply:
version name is overridden by fastlane https://github.com/mosip/inji-wallet/blob/develop/android/fastlane/Fastfile#L77